<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finditem_admin</class>
 <widget class="QDialog" name="finditem_admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;üîé –ü–æ–∏—Å–∫ –ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏—è üîé&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton { background-color: #F5F2E7; /* –Ø—Ä–∫–æ-–∫—Ä–∞—Å–Ω—ã–π */ color: black; /* –ß–µ—Ä–Ω—ã–π —Ç–µ–∫—Å—Ç */ } QPushButton:hover { background-color: #D32F2F; /* –¢–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */ } QPushButton:pressed { background-color: #B71C1C; /* –ï—â—ë —Ç–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */ }
</string>
   </property>
   <property name="text">
    <string>–í–µ—Ä–Ω—É—Ç—å—Å—è –Ω–∞–∑–∞–¥</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>691</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>651</width>
      <height>284</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ñ–∞–Ω—Ä&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="section">
       <property name="placeholderText">
        <string> –í–≤–µ–¥–∏—Ç–µ –∂–∞–Ω—Ä –ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏—è</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ê–≤—Ç–æ—Ä&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="author">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –∞–≤—Ç–æ—Ä–∞ –ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏—è</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ù–∞–∑–≤–∞–Ω–∏–µ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –Ω–∞–∑–≤–∞–Ω–∏–µ –ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏—è</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ò–∑–¥–∞—Ç–µ–ª—å&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="publisher">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –∏–∑–¥–∞—Ç–µ–ª—è —ç–∫–∑–µ–º–ø–ª—è—Ä–∞</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ì–æ–¥ –≤—ã–ø—É—Å–∫–∞&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="publisher_year">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –≥–æ–¥ –≤—ã–ø—É—Å–∫–∞ —ç–∫–∑–µ–º–ø–ª—è—Ä–∞</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ö–æ–ª–∏—á–µ—Å—Ç–≤–æ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="quantity">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –∫–æ–ª–∏—á–µ—Å—Ç–≤–æ —ç–∫–∑–µ–º–ø–ª—è—Ä–æ–≤</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–¶–µ–Ω–∞&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="price">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ —Ü–µ–Ω—É —ç–∫–∑–µ–º–ø–ª—è—Ä–∞</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–î–æ–ø. –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="additional_info">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –¥–æ–ø. –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—é —ç–∫–∑–µ–º–ø–ª—è—Ä–∞</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="dfgdgf">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ù–æ–º–µ—Ä –º–∞–≥–∞–∑–∏–Ω–∞&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="numberShop">
       <property name="placeholderText">
        <string>–í–≤–µ–¥–∏—Ç–µ –Ω–æ–º–µ—Ä –º–∞–≥–∞–∑–∏–Ω–∞, –≥–¥–µ —Ö—Ä–∞–Ω–∏—Ç—Å—è —ç–∫–∑–µ–º–ø–ª—è—Ä</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #F5F2E7; /* –Ø—Ä–∫–æ-–∫—Ä–∞—Å–Ω—ã–π */
    color: black;              /* –ß–µ—Ä–Ω—ã–π —Ç–µ–∫—Å—Ç */
 
}

QPushButton:hover {
    background-color: #01eb00; /* –¢–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */
}

QPushButton:pressed {
    background-color: #01a800 /* –ï—â—ë —Ç–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */
}



</string>
   </property>
   <property name="text">
    <string>–ù–∞–π—Ç–∏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
